frontend/packages/ceph-storage-plugin/src/types.ts

```diff
@@ -246,17 +246,20 @@ export const VaultAuthMethodMapping: {
   [keys in VaultAuthMethods]: {
     name: string;
     value: VaultAuthMethods;
+    isWizardFlowOnly: boolean;
     supportedEncryptionType: KmsEncryptionLevel[];
   };
 } = {
   [VaultAuthMethods.KUBERNETES]: {
     name: 'Kubernetes',
     value: VaultAuthMethods.KUBERNETES,
+    isWizardFlowOnly: true,
```

Comment:
Do we really need this prop to be passed? Can we figure out from the location that we are in the wizard flow or not?